packages/ui/src/components/Button/Button.tsx

```diff
@@ -25,23 +26,35 @@ export function Button({
             ref={forwardRef}
             type={type}
             className={twMerge(
-                'rounded flex items-center justify-center border border-transparent bg-transparent',
-                size === 'small' ? `p-3 label-medium` : `p-4 label-large`,
+                `${
+                    rounded ? 'rounded-full' : 'rounded'
+                } flex items-center justify-center border border-transparent bg-transparent`,
+                size === 'small'
+                    ? `${rounded ? 'py-2 px-3' : 'p-3'} label-medium`
+                    : `${rounded ? 'py-3 px-4' : 'p-4'} label-large`,
```

Comment:
I know this is by Figma design, but it feels weird that spacing is different for rounded buttons.
I'd consult about that with Hugo and Al. Ideally we'd want the spacing to be uniform. If we need extra variations, it's better to add more sizing variants.